tooling/src/main/java/org/opencds/cqf/tooling/acceleratorkit/Processor.java

```diff
@@ -3829,6 +3832,27 @@ public void writeDataElements(String scope, String scopePath, String context) {
         }
     }
 
+    private String retrieveLibraryShortName(){
+        switch (commonLibraryName) {
+            case "NACHCCommon":
+                return "NC";
+            case "WHOCommon":
+                return "WC";
+            case "SDHCommon":
+                return "SC";
+            default:
+                return "";
+        }
+    }
+    private ArrayList<String> retrieveProfileLibrary() {
+        for(String profileKey: this.profilesByParentProfile.keySet()){
+            if(profileKey.contains("qicore")){
+                return new ArrayList<String>(Arrays.asList("using QICore version '4.1.1'", "include QICoreCommon called QC"));
+            }
+        }
+        return new ArrayList<String>(Arrays.asList("using FHIR version '4.0.1'", "include FHIRCommon version '1.1.000' called FC"));
+    }
+
     public void writeIgJsonFragments(String path) {
         try (FileOutputStream writer = new FileOutputStream(path + "/ig.json")) {
             writer.write(String.format("{\r\n%s\r\n}", String.join(",\r\n", igJsonFragments)).getBytes());
```

Comment:
Use the format built-in codes so that the result is not platform specific, ex: \n => %n.